main.go

```diff
@@ -11,12 +13,45 @@ import (
 	"github.com/scraperwiki/hookbot/pkg/listen"
 )
 
+var slackUrl string
+
+type SlackMessage struct {
+	Text      string `json:"text"`
+	Username  string `json:"username"`
+	IconEmoji string `json:"icon_emoji"`
+	Channel   string `json:"channel"`
+}
+
+func SendToSlack(eventData []byte) (err error) {
```

Comment:
Repeating this since I accidentally commented on the commit and not the PR:

Since this is used asynchronously, there is nowhere to send the error (i.e, `go SendToSlack()`). We will at some point do strict linting and require all errors are checked, so this would become an error, because `SendToSlack` returns an error value that no-one looks at. Since it serves no purpose in this case, you can just remove the return value and simply `return`.